ui/raidboss/data/06-ew/raid/p7n.ts

```diff
@@ -1,13 +1,172 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  busterTargets: string[];
+}
 
 const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.AbyssosTheSeventhCircle,
   timelineFile: 'p7n.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      busterTargets: [],
+    };
+  },
+  timelineTriggers: [
+    {
+      id: 'P7N Burst',
+      regex: /Burst/,
+      beforeSeconds: 4,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get towers',
+        },
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'P7N Bough Of Attis Out',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '77F9', source: 'Agdistis', capture: false }),
+      response: Responses.getOut(),
+    },
+    {
+      id: 'P7N Bough Of Attis In',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '77FE', source: 'Agdistis', capture: false }),
+      response: Responses.getIn(),
+    },
+    {
+      id: 'P7N Bough Of Attis Left',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '77FC', source: 'Agdistis', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'P7N Bough Of Attis Right',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '77FB', source: 'Agdistis', capture: false }),
+      response: Responses.goRight(),
+    },
+    {
+      id: 'P7N Hemitheos Holy Spread',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0137' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'P7N Hemitheos Glare',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '79FA', source: 'Agdistis', capture: false }),
+      suppressSeconds: 5,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Move center when safe',
+        },
+      },
+    },
+    {
+      id: 'P7N Immortals Obol',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '77F5', source: 'Agdistis', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'In circle--get to edge',
```

Comment:
Sorry, can you remind me what this ability is? I'm not sure I understand this callout.

Reply:
This is "hey, the floor is falling, get to one of the three safespots and away from the center". Because the falloff damage is logarithmic just like Lightning Mekhane, I figured that it was best to specify being at the arena's edge.

Reply:
Maybe the reverse makes more sense? "Get to edge (in circle)"